linux/base.Dockerfile

```diff
@@ -111,12 +110,21 @@ RUN tdnf update -y && bash ./tdnfinstall.sh \
   zsh
 
 # Install Maven
-RUN tdnf update -y && bash ./tdnfinstall.sh maven
-
-RUN tdnf clean all
+RUN bash ./tdnfinstall.sh \
```

Comment:
Can't we put autoconf, ansible in the above section here?  Just curious.

Reply:
Also, NodeJs. This way, we will have all the tdnfintall.sh packages are at one place.